tests/test_integration.py

```diff
@@ -1130,6 +1130,34 @@ def test_pex_source_bundling():
             assert stdout == b"hello\n"
 
 
+def test_pex_source_bundling_pep420():
+    with temporary_dir() as output_dir:
+        with temporary_dir() as input_dir:
+            with safe_open(os.path.join(input_dir, "a/b/c.py"), "w") as fh:
```

Comment:
Nit, you can combine these to reduce nesting:

```python
with temporary_dir() as output_dir, temporary_dir() as input_dir():
```

This is fine to keep - only want to make sure you're aware.

Reply:
Ack. I'll stick with the copypasta from the test above for now.